solutions/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from .views import *
+
+app_name = 'solutions'
+
+urlpatterns = [
+    path('list/<int:mission_id>/', SolutionHistoryListView.as_view(), name='solution-list'),
+    path('detail/<str:solution_history_id>/', SolutionHistoryDetailView.as_view(), name='solution-detail'),
+    path("create/", SolutionHistoryCreateView.as_view(), name="solution-create"),
+    path("update/<str:solution_history_id>/", SolutionHistoryUpdateView.as_view(), name="solution-update"),
```

Comment:
💊 클라이언트 요청의 Method에 따라 자동으로 적절한 메서드를 실행하는 APIView의 특징을 살려보는 건 어떨까요?
```suggestion
    path('', Root.as_view()), # 추가
    path('<int:mission_id>', Missionid.as_view()), # 목록 조회
    path('<str:solution_history_id>', Solutionhistoryid.as_view()), # 상세 조회, 수정
```